mailerlite/group.py

```diff
@@ -205,6 +205,48 @@ def add_subscribers(self, group_id, subscribers_data, resubscribe=False,
 
         return [Subscriber(**subs) for subs in res_json['imported']]
 
+    def add_single_subscriber(self, group_id, subscribers_data: dict,
+                              resubscribe=False, autoresponders=False,
+                              as_json=False):
+        """Add single new subscriber to specified group.
+
+        https://developers.mailerlite.com/v2/reference#add-single-subscriber
+
+        Parameters
+        ----------
+        group_id : int
+            group id
+        subscribers_data : dict,
+            subscribers data
+        resubscribe : bool
+            reactivate subscriber if value is true (default False)
+        autoresponders : bool
+            autoresponders will be sent if value is true (default False)
+        as_json : bool
+            return result as json format
+
+        Returns
+        -------
+        subscriber: :class:Subscriber
+            subscriber object
+
+        """
+        url = client.build_url('groups', group_id, 'subscribers')
+
+        body = {'resubscribe': resubscribe, 'autoresponders': autoresponders,
+                **subscribers_data}
+
+        if not {'email', 'name'}.issubset(subscribers_data.keys()):
```

Comment:
I just learned something, `issubset` seems really useful! 👍🏾

Reply:
Yay! And it's faster than checking if both keys are in dict.keys() like 
`'email' in subscribers_data.keys()`